scrum/tools/sprint_math.py

```diff
@@ -222,38 +247,47 @@ class SprintMath:
             Validates the user input to ensure it is a valid integer.
 
         calc_planned_next_sprint(self):
-            Calculates the target planned points for the next sprint. Assigns result to sp_next_sprint of current object.
+            Calculates the target planned points for the next sprint. 
+                Assigns result to sp_next_sprint of current object.
     """
-    def __init__(self, sp_unplanned_total = 0, sp_unplanned_remaining = 0, sp_unplanned_partial_completed = 0,
-                 total_done_list = 0, sp_planned_partial_completed = 0, sp_retro_completed = 0,
-                 sp_retro_leftover = 0, unplanned_past_sprints = 0):
+    def __init__(self, gathered_sp_unplanned_total = 0,
+                 gathered_sp_unplanned_remaining = 0,
+                 gathered_sp_unplanned_partial_completed = 0,
+                 gathered_total_done_list = 0,
+                 gathered_sp_planned_partial_completed = 0,
+                 gathered_sp_retro_completed = 0,
+                 gathered_sp_retro_leftover = 0,
+                 gathered_unplanned_past_sprints = 0):
         # Assigning everything captured for calculations later on
-        self.sp_unplanned_total = sp_unplanned_total
-        self.sp_unplanned_remaining = sp_unplanned_remaining
-        self.sp_unplanned_partial_completed = sp_unplanned_partial_completed
-        self.total_done_list = total_done_list
-        self.sp_planned_partial_completed = sp_planned_partial_completed
-        self.sp_retro_completed = sp_retro_completed
-        self.sp_retro_leftover = sp_retro_leftover
-        self.unplanned_past_sprints = unplanned_past_sprints
+        self.sp_unplanned_total = gathered_sp_unplanned_total
```

Comment:
They were renamed because pylint didn't like that it was being redefined (even though the ones on the left belong to the class). I think this is fixed now though